formspree/forms/models.py

```diff
@@ -42,6 +60,16 @@ def __init__(self, email, host):
     def __repr__(self):
         return '<Form %s, email=%s, host=%s>' % (self.id, self.email, self.host)
 
+    def get_random_like_string(self):
+        if not self.id:
+            raise Exception("this form doesn't have an id yet, commit it first.")
+        return int2bigstring(self.id)
```

Comment:
The way I imagined it, we could just use the primary id of the form as the target endpoint, as a way to enable hide the target email from scrapers only. The random_like_string was always imagined as just an aesthetic thing. `int2bigstring` was never meant to encrypt or hide anything from anyone, as it is obvious.

The confirmation process still uses a hash involving the secret NONCE, just like the confirmation emails for normal forms. See lines 161 and 197 of this same file. Also the explanation at 21-36.